pkg/acs/calacs/lib/getacskeys.c

```diff
@@ -46,14 +46,23 @@ int GetACSKeys (ACSInfo *acs, Hdr *phdr) {
   
 	if (GetKeyStr (phdr, "ROOTNAME", NO_DEFAULT, "",
                  acs->rootname, ACS_CBUF))
-    return (status);
+       return (status);
   
 	if (GetKeyStr (phdr, "APERTURE", USE_DEFAULT, "", acs->aperture, ACS_CBUF))
-    return (status);
+       return (status);
+    {unsigned int i;
```

Comment:
This should be a func ``toUpper()`` that excepts a string. E.g. use this [utility in cfitsio](https://heasarc.gsfc.nasa.gov/docs/software/fitsio/c/c_user/node52.html) ``Convert a character string to uppercase (operates in place).
  void fits_uppercase / ffupch (char *string)``.